content/cpp/concepts/math-functions/terms/fmod/fmod.md

```diff
@@ -0,0 +1,94 @@
+---
+Title: 'fmod()'
+Description: 'Computes the floating point remainder of numerator/denominator (rounded towards zero)'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`fmod()`** function in C++ computes the floating point remainder of numerator/denominator (rounded towards zero).
+
+## Syntax
+
+```pseudo
+fmod (x, y)
+```
+
+x: The value of numerator,
+y: The value of denominator.
+
+## Prototype [C++11 Standard]
+
+```cpp
+double fmod(double x, double y);
+float fmod(float x, float y);
+long double fmod(long double x, long double y);
+// Additional overloads
+double fmod(Type1 x, Type2 y);
+```
+
+## Definition and Return Type
+
+Additional overloads are provided in this header (<cmath>) for other combinations of arithmetic types (Type1 and Type2): These overloads effectively cast its arguments to double before calculations, except if at least one of the arguments is of type long double (in which case both are casted to long double instead).
```

Comment:
"~These~ The ~overloads~ overloaded functions ~effectively~ cast ~its~ the arguments to ~double~ a `double` type before ~calculations~ the calculation ~,~. ~except~ If ~at least~ one of the arguments is of type ~long double~ `long double` ~(in which case both are casted to long double instead)~, both arguments are cast to a `long double` type"